lib/sandboxed_renderer/init.js

```diff
@@ -20,19 +20,23 @@ for (let prop of Object.keys(events.EventEmitter.prototype)) {
 }
 Object.setPrototypeOf(process, events.EventEmitter.prototype)
 
-const electron = require('electron')
-const fs = require('fs')
-const preloadModules = new Map([
-  ['child_process', require('child_process')],
-  ['electron', electron],
-  ['events', events],
-  ['fs', fs],
-  ['os', require('os')],
-  ['path', require('path')],
-  ['url', require('url')],
-  ['timers', require('timers')]
+const preloadModules = new Set([
+  'child_process',
+  'electron',
+  'fs',
```

Comment:
why did you remove events?

Reply:
It was added recently in https://github.com/electron/electron/pull/12828

Reply:
It was a merge error, I took this patch from an older electron version we use in our app. Will add it back :+1: